tests/components/forked_daapd/test_media_player.py

```diff
@@ -0,0 +1,559 @@
+            "volume": 0,
+        },
+        {
+            "id": "0",
+            "name": "Computer",
+            "type": "ALSA",
+            "selected": True,
+            "has_password": False,
+            "requires_auth": False,
+            "needs_auth_key": False,
+            "volume": 19,
+        },
+        {
+            "id": "100",
+            "name": "daapd-fifo",
+            "type": "fifo",
+            "selected": False,
+            "has_password": False,
+            "requires_auth": False,
+            "needs_auth_key": False,
+            "volume": 0,
+        },
+    ]
+
+
+@pytest.fixture(name="sample_outputs_off")
+def sample_outputs_off_fixture():
+    """Create outputs fixture."""
+    return [
+        {
+            "id": "123456789012345",
+            "name": "kitchen",
+            "type": "AirPlay",
+            "selected": False,
+            "has_password": False,
+            "requires_auth": False,
+            "needs_auth_key": False,
+            "volume": 0,
+        },
+        {
+            "id": "0",
+            "name": "Computer",
+            "type": "ALSA",
+            "selected": False,
+            "has_password": False,
+            "requires_auth": False,
+            "needs_auth_key": False,
+            "volume": 19,
+        },
+        {
+            "id": "100",
+            "name": "daapd-fifo",
+            "type": "fifo",
+            "selected": False,
+            "has_password": False,
+            "requires_auth": False,
+            "needs_auth_key": False,
+            "volume": 0,
+        },
+    ]
+
+
+@pytest.fixture(name="device")
+async def device_fixture(
+    hass,
+    sample_config,
+    sample_player_paused,
+    sample_queue,
+    sample_outputs_on,
+    sample_tts_queue,
+):
+    """Create device fixture."""
+    # queue_iter = itertools.cycle([sample_queue, sample_tts_queue])
+    # sample_player = [sample_player_paused, sample_player_playing]
+    # sample_outputs = [sample_outputs_on, sample_outputs_off]
+
+    async def get_request_side_effect(update_type):
+        if update_type == "player":
+            return sample_player_paused
+        if update_type == "config":
+            return sample_config
+        if update_type == "outputs":
+            return {"outputs": sample_outputs_on}
+        if update_type == "queue":
+            return sample_tts_queue
+
+    def async_add_entities(list_of_entities, update_on_add):
+        for entity in list_of_entities:
+            print(entity)
+        print(update_on_add, "Update on add: {update_on_add}")
+
+    async def add_to_queue_side_effect(uris, playback, playback_from_position=None):
+        await device._update_queue()  # sorry for touching private
+        await device._update_player()  # sorry for touching private
+
+    with patch(
+        "homeassistant.components.forked_daapd.media_player.ForkedDaapdAPI",
+        autospec=True,
+    ):
+        device = ForkedDaapdDevice(
```

Comment:
Don't create the entity directly. Set up the integration and platform and let it add an entity. Then assert the state of the entity via the core state machine.

See eg:
https://github.com/home-assistant/core/blob/4484afc0d6756f2df140925285226cd2d1eb1e13/tests/components/heos/test_media_player.py#L66-L100